src/toast/_libtoast/ops_pixels_healpix.cpp

```diff
@@ -1,39 +1,62 @@
-// Copyright (c) 2015-2021 by the parties listed in the AUTHORS file.
+// Copyright (c) 2015-2023 by the parties listed in the AUTHORS file.
 // All rights reserved.  Use of this source code is governed by
 // a BSD-style license that can be found in the LICENSE file.
 
-#include <module.hpp>
+// All compiled healpix functionality should be in this file, and it
+// must be a single compilation unit.
 
+#include <module.hpp>
 #include <qarray.hpp>
-
 #include <intervals.hpp>
-
 #include <accelerator.hpp>
 
+// PI
+#define PI 3.14159265358979323846
```

Comment:
This seems a little dated, given that mathematical constants are available through standard libraries but since it works I would not worry about it.

Reply:
True, I could include `cmath` for some of them- that would be a small change.  Although there may still be some differences between systems.  [Looks like C++20 finally adds them](https://stackoverflow.com/questions/1727881/how-to-use-the-pi-constant-in-c), so someday it will be easier.

Reply:
I will make this change in #702 instead.